public/code-map/graphify-out/GRAPH_REPORT.md

```
@@ -0,0 +1,249 @@
+
+### Community 30 - "Community 30"
+Cohesion: 0.40
+Nodes (3): Props, Props, TitleBarButton
+
+### Community 31 - "Community 31"
+Cohesion: 0.33
+Nodes (5): ARTICLE_EXPLORERS, EVENT_EXPLORERS, ExplorerItem, ExplorerLink, PROFILE_EXPLORERS
+
+### Community 32 - "Community 32"
+Cohesion: 0.40
+Nodes (4): compat, __dirname, eslintConfig, __filename
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)